phalcon/dispatcherinterface.zep

```diff
@@ -125,8 +123,6 @@ interface DispatcherInterface
 
 	/**
 	 * Forwards the execution flow to another controller/action
-	 *
-	 * @param array forward
 	 */
-	public function forward(forward);
+	public function forward(array forward);
```

Comment:
From a technical perspective yes -- however, the subsequent line was throwing an error if not an array. So both throw an array. Therefore, from an implementation perspective, there is no difference.